bundle/jsky.app.ot/src/main/scala/jsky/app/ot/tpe/ImageCatalogPanel.scala

```diff
@@ -111,19 +112,28 @@ final class ImageCatalogPanel(imageDisplay: CatalogImageDisplay) {
     }
   }
 
-  lazy val panel: Component = new MigPanel(LC().fill().insets(0.px).gridGap(0.px, 0.px)) {
-    add(new Label("Image Catalog:"), CC())
-    add(toolsButton, CC().alignX(RightAlign))
-    catalogRows.foreach { row =>
-      add(row.button, CC().newline())
-      add(row.feedback, CC().alignX(RightAlign))
-      buttonGroup.add(row.button.peer)
-      row.button.reactions += {
-        case ButtonClicked(_) =>
-          requestImage(row.feedback.catalog)
-        }
+  private def row(l: Component, r: Component): Component =
+    new BoxPanel(Orientation.Horizontal) {
+      contents ++= List(l, Swing.HGlue, r)
+    }
+
+  lazy val panel: Component =
+    new BorderPanel() {
+      add(row(new Label("Image Catalog"), toolsButton), North)
+      add(Swing.HStrut(5), West)
+      add(
+        new GridPanel(0, 2) {
+          contents ++= catalogRows.map { r => row(r.button, r.feedback) }
+          catalogRows.foreach { r =>
+            buttonGroup.add(r.button.peer)
+            r.button.reactions += {
+              case ButtonClicked(_) => requestImage(r.feedback.catalog)
+            }
+          }
+        },
+        Center
+      )
```

Comment:
This should probably be done with the Mig layout library since the rest of it was done that way.  Suggestions for how to do that or PRs editing this are welcome.